docs/3.0rc/develop/write-tasks/use-task-runners.mdx

```diff
@@ -100,34 +97,26 @@ Prefect uses the default `ConcurrentTaskRunner`.
 ## Running tasks sequentially
 
 Sometimes, it's useful to force tasks to run sequentially to make it easier to reason about the 
-behavior of your program. Switching to the `SequentialTaskRunner` will force submitted tasks to run 
-sequentially rather than concurrently.
+behavior of your program. Calling your tasks directly without using `.submit()` will automatically execute your tasks
+in order.
 
-<Note>
```

Comment:
perhaps we could keep an admonition like "`SequentialTaskRunner` is longer necessary, just call your task" or link to task concept page since it seems likely that people will have used it for one reason or another and come here looking for it after installing 3.x / breaking imports